### PR TITLE
Add TB app to packages

### PR DESCRIPTION
**Pull request by Kasulejoseph**

### CODE REVIEW

**Comment by Kasulejoseph** [comment]
> @Kasulejoseph let's clean the TB package by removing none tb stuff i.e forms etc

Alright

File: packages/esm-tb-app/src/routes.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://json.openmrs.org/routes.schema.json",
+  "backendDependencies": {
+    "fhir2": "^1.2.0",
+    "webservices.rest": "^2.24.0"
+  },
+  "pages": [],
+  "extensions": [
+    {
+      "component": "opdDashboardLink",
```
Comment (jabahum) [inline packages/esm-tb-app/src/routes.json:10]: can we have this changed to the right module names

File: packages/esm-tb-app/src/routes.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "name": "opd-dashboard",
+      "slot": "patient-chart-dashboard-slot",
+      "online": true,
+      "offline": true
+    },
+    {
+      "component": "opdDashboardExt",
+      "name": "opd-dashboard-ext",
```
Comment (jabahum) [inline packages/esm-tb-app/src/routes.json:18]: this too

**Review by jabahum — CHANGES_REQUESTED** [review]

**Comment by jabahum** [comment]
@Kasulejoseph make sure to run yarn verify to help resolve the CI errors

File: packages/esm-tb-app/translations/en.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (jabahum) [inline packages/esm-tb-app/translations/en.json:1]: Do we need opd here

**Review by jabahum — APPROVED** [review]

**Pull request merged**